web/tests/hotkey.test.cjs

```diff
@@ -355,11 +357,11 @@ function test_normal_typing() {
     assert_unmapped('~!@#$%^*()_+{}:"<>');
 }
 
-test_while_not_editing_text("unmapped keys return false easily", () => {
+run_test("unmapped keys return false easily", () => {
     // Unmapped keys should immediately return false, without
-    // calling any functions outside of hotkey.js.
+    // calling any functions outside of hotkey.ts.
     // (unless we are editing text)
-    assert_unmapped("bfoyz");
+    assert_unmapped("bfo");
```

Comment:
After updating `selected_message` to actually have a message type, `y` and `z` do something